content/docs/minion-v0.4/tag_cache.md

```diff
@@ -105,7 +105,8 @@ Nodes from all tree levels use the same size so they can be stored and searched
 This also allows dynamic space allocation between the table nodes and the map nodes.
 When most memory words are tagged, map nodes can be replaced to store table nodes;
 when most memory words are not tagged, the cache will primarily hold map
-nodes.
+nodes. The default block size is 512 bits, however 64 bits is more optimal.
```

Comment:
I think what I said in my email is I prefer to say a cache line is 64 bytes rather than 512 bits.